### PR TITLE
re #808 SEO Improvements

### PR DESCRIPTION
**Pull request by kahboom**
Changes:
- Updated title format for all blog posts to include primary keyword.
- Updated title for all static pages on the website.
- Added `rel="nofollow"` and `target="_blank"` for external links.
- Added Jekyll sitemap generator.
- Added Bing meta tag for validation.
- Added some image `alt` attributes to be descriptive.
- Fixed a minor typo in a blog post.

JIRA: https://issues.jboss.org/browse/APIMAN-808

We will need to keep monitoring Google Analytics and other tools to see how we perform with those keywords and with these changes. We can go adjusting these changes accordingly. Monitoring should be continuous!

cc @EricWittmann

### CODE REVIEW

**Comment by EricWittmann** [comment]
Note: a couple of the files you changed are actually generated - so the SEO changes you made will get whacked next time they are generated.  Hit me up on IRC tomorrow if you think it's something we should address.

File: _blog-src/_posts/2016-01-11-release-1.2.markdown [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title:  "Finall!  Apiman 1.2.0.Final is released!"
+title:  "Final!  Apiman 1.2.0.Final is released!"
```
Comment (kahboom) [inline _blog-src/_posts/2016-01-11-release-1.2.markdown]: In retrospect, I think this should have been 'Finally!' not 'Final!' right @EricWittmann ? Let me know and I'll update it.

File: latest/apidoc/manager/index.html [hunk 2]
```diff
@@ -375,14 +363,13 @@
...
-                                As such, not all features are supported.
-                                <span set-html="visitWebsiteForProVersion"></span>
+                                <i class="icon-info-sign" title="Free version"></i> Notice: This documentation was
+                                generated by the free version of MireDot. As such, not all features are supported. <span
+                                    set-html="visitWebsiteForProVersion"></span>
                             </div>
                             <div class="notice invalidlicense" set-if="!validLicense">
-                                <i class="icon-exclamation-sign" title="Invalid key"></i>
-                                <span set-html="licenseErrorMessage"></span>
+                                <i class="icon-exclamation-sign" title="Invalid key"></i> <span
```
Comment (kahboom) [inline latest/apidoc/manager/index.html]: No idea what's up with these spacing issues.

**Comment by kahboom** [comment]
Thanks @EricWittmann - The ones that I noticed were automatically generated after I had already started making changes were `latest/installation-guide/index.html` and `latest/production-guide/index.html`, which is totally fine. Were there any others that I should be aware of?

**Pull request merged**